components/dashboards-web-component/src/utils/apis/DashboardAPI.jsx

```diff
@@ -144,4 +144,16 @@ export default class DashboardAPI {
             .getHTTPClient()
             .get(`/report-config`);
     }
+
+    /**
+     * Export Dashboard
+     *
+     * @param {string} dashboardId Dashboard ID
+     * @returns {Promise} Promise
+     */
+    static handleExportDashboard(dashboardId) {
```

Comment:
This function does not "handle" dashboard export, so we need to find a better name. Also, IMO this function should be at the `DashboardAPI` class. See https://github.com/wso2/carbon-dashboards/blob/master/components/dashboards-web-component/src/utils/apis/DashboardAPI.jsx#L116

Reply:
Fixed in #5066ee8